test/e2e/src/tests/test_e2e_workflows.py

```diff
@@ -120,6 +120,7 @@ def test_e2e_workflows(env_dict):
         pytest.fail("rowCount should be greater than for cluster: {0} for workflow: {1} ".format(clusterResourceId, 'KUBE_SERVICES'))
 
     # KubeEvents
+    # only the kube events with !normal event type will be collected
```

Comment:
This test failing because there are no !normal events , right? we would need to make the change in helm chart to set this environment variable AZMON_CLUSTER_COLLECT_ALL_KUBE_EVENTS=true in Test mode on ama-logs-rs pod which is similar to this https://github.com/microsoft/Docker-Provider/blob/ci_prod/charts/azuremonitor-containers/templates/ama-logs-daemonset.yaml#L135, right?

Reply:
it is added: https://github.com/microsoft/Docker-Provider/blob/9f16b604bb7a453dc4ba95962e43902a311ddf6d/charts/azuremonitor-containers/templates/ama-logs-deployment.yaml#L118